src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+$square: 100px;
```

Comment:
add all variables to new file _variables.scss, you can do it in folder 'utils' inside of folder 'styles'

Reply:
i dont know why, but "main" cant find "_variables".